### PR TITLE
Swift3.0-Swift2.3 Compatability

### PR DESCRIPTION
**Pull request by dostrander**
- Change names to swift3 style
- Internalize instead of privateize

### CODE REVIEW

**Comment by KingOfBrian** [comment]
Just a heads up, @nevillco is working on a PR with horizontal / vertical type information which will also merge into @ahtierney branch, so keep him in the loop!

**Comment by dostrander** [comment]
e@KingOfBrian will do. @nevillco let me know if you've already done this and i should just close it. Or if you haven't you can just merge it in

**Comment by nevillco** [comment]
Hey @dostrander yeah I already pushed my changes to @ahtierney branch. Thanks!

**Comment by nevillco** [comment]
@dostrander You can revert my commit that got in this branch - opening up a separate PR for my story

File: Anchorage.swift [hunk 1]
```diff
@@ -33,9 +33,13 @@ extension NSLayoutDimension : LayoutAnchorType {}
...
 extension NSLayoutYAxisAnchor : LayoutAnchorType {}
 
+public protocol LayoutAxisType {}
+extension NSLayoutXAxisAnchor : LayoutAxisType {}
+extension NSLayoutYAxisAnchor : LayoutAxisType {}
+
 #if swift(>=3.0)
 
-// MARK: - Equality Constraints
+    // MARK: - Equality Constraints
```
Comment (ZevEisenberg) [inline Anchorage.swift]: 😱

**Comment by KingOfBrian** [comment]
@ahtierney is going to merge this into his branch by hand, I'm going to close this PR. Thanks @dostrander !

**Pull request closed without merging**